src/updater.cc

```diff
@@ -58,18 +59,22 @@ PollingMetadataUpdater::~PollingMetadataUpdater() {
   }
 }
 
-bool PollingMetadataUpdater::ValidateConfiguration() const {
-  return period_ >= time::seconds::zero();
+bool PollingMetadataUpdater::ValidateConfiguration() const
+    throw(ValidationError) {
+  if (period_ < time::seconds::zero()) {
+    throw ValidationError(
+        format::Substitute("Polling period {{period}}s cannot be negative",
+                           {{"period", format::str(period_.count())}}));
+  }
+  return period_ > time::seconds::zero();
```

Comment:
Let's change this to >= again, as we will have a new abstraction for "ShouldStartUpdater" that will focus strictly on > 0.

Reply:
Actually, just moved this code to `ShouldStartUpdater` — `ValidateConfiguration` no longer needs a return value.